metricflow/model/parsing/schemas/metricflow.json

```diff
@@ -713,10 +713,10 @@
         "data_source": {
             "$ref": "#/definitions/data_source"
         },
-        "derived_group_by_element_schema": {
+        "derived_identifier": {
```

Comment:
I think we can remove "derived_identifier" (and probably should as I don't believe it's currently supported)

Reply:
I think I will leave it here for now until we strip it all out of the code.